src/discord.ts

```diff
@@ -69,7 +74,37 @@ export async function sendDailyStatsToDiscord(params: DailyStatsParams) {
 interface FlaggedRunsParams {
   sm64RecentlyExamined: ExaminedRun[];
 }
+interface RejectedRunParams {
+  rejectedRun: Run;
+  rejectionFlags: Flag[];
+}
+export async function sendRejectedRunToDiscord(params: RejectedRunParams) {
+  const {rejectedRun, rejectionFlags} = params;
+  const user = await getPlayerFromRun(rejectedRun);
```

Comment:
nice, i like getting the player name for runs. instead of only using it here, can you move this into `runToString` so that it now returns the user name with it? that way our flagged runs notice in the main chan will show usernames too